pubsub/stanpubsub/stan.go

```diff
@@ -0,0 +1,538 @@
+// Copyright 2021 The Go Cloud Development Kit Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package stanpubsub provides a pubsub implementation for NATS Streaming client.
```

Comment:
Can you add a link for NATS Streaming here?

Reply:
Done.